src/chains/ethereum/options/src/logging-options.ts

```diff
@@ -92,23 +101,48 @@ export const LoggingOptions: Definitions<LoggingConfig> = {
     cliAliases: ["q", "quiet"],
     cliType: "boolean"
   },
-  logger: {
-    normalize,
-    cliDescription:
-      "An object, like `console`, that implements a `log` function.",
-    disableInCLI: true,
-    // disable the default logger if `quiet` is `true`
-    default: config => ({
-      log: config.quiet ? () => {} : console.log
-    }),
-    legacyName: "logger"
-  },
   verbose: {
     normalize,
     cliDescription: "Set to `true` to log detailed RPC requests.",
     default: () => false,
     legacyName: "verbose",
     cliAliases: ["v", "verbose"],
     cliType: "boolean"
+  },
+  file: {
+    normalize: (raw: PathLike): number => {
+      let descriptor: number;
+
+      try {
+        descriptor = openSync(raw, "a");
+      } catch (err) {
+        throw new Error(
+          `Failed to open log file ${raw}. Please check if the file path is valid and if the process has write permissions to the directory.`
+        );
+      }
+      return descriptor;
+    },
+    cliDescription:
+      "If set, Ganache will write logs to a file located at the specified path.",
```

Comment:
Maybe something like `The filename to append logs to.`?

Reply:
That's certainly less convoluted than my description :P